### PR TITLE
Reworded the article about profiler storage

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 3.0+ |
| Fixed tickets | - |

This finishes #6024.

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Can we consider this PR as "finished"? Thanks.

**Comment by javiereguiluz** [comment]
Could we consider this PR as "finished" to merge it soon? Some people are confused by the current outdated contents (see https://twitter.com/SimonMinotto/status/694824051232108544)  Thanks!

File: cookbook/profiler/storage.rst [hunk 1]
```diff
@@ -4,66 +4,21 @@
 Switching the Profiler Storage
 ==============================
 
-By default the profile stores the collected data in files in the ``%kernel.cache_dir%/profiler/`` directory.
-You can control the storage being used through the ``dsn``, ``username``,
-``password`` and ``lifetime`` options. For example, the following configuration
-uses MySQL as the storage for the profiler with a lifetime of one hour:
+In Symfony versions previous to 3.0, profiles could be stored in files, databases,
```
Comment (WouterJ) [inline cookbook/profiler/storage.rst]: "previous to" should be "prior to" to be consistent.

**Comment by WouterJ** [comment]
:+1:

**Comment by xabbuh** [comment]
Thank you @xavierleune and @javiereguiluz! I merged this into the `3.0` branch and made some tweaks in 35a520dfc38af4a3f1273efd842c7e1f7411d546.

**Pull request merged**